frontend/getLabels.py

```diff
@@ -3,7 +3,6 @@
 labels=''
 rawLabels=json.loads(sys.argv[1])['labels']
 for rLabel in rawLabels:
-  if rLabel['name'].startswith("kiali-bot"):
```

Comment:
seems odd we would just remove this outer if-stmt conditional and yet leave the remaining inner if-stmts as-is. Seems like this would change behavior.

@aljesusg you created this "getLabels.py" file way back in 2018 and it hasn't been touched since. Do you know if it is even used anymore? I don't see where it is used. My question is: can we just delete this frontend/getLabels.py file entirely?

Reply:
Well this was a requirement from qe to check the labels of the PR. I think that is deprecated because we are not using it so we can remove it.

Reply:
@FreelancingTwin as per discussion above, we can delete this `frontend/getLabels.py` file entirely.